docker-compose/swe-idcc/config/data-bridge/swe-idcc-boundary-eq-data-bridge-application.yml

```diff
@@ -0,0 +1,24 @@
+data-bridge:
+  zone-id: "Europe/Paris"
+  target-process: SWE_IDCC
+  file-type: BOUNDARY_EQ
+  file-regex: (?<year>[0-9]{4})(?<month>[0-9]{2})(?<day>[0-9]{2})_(?<hour>[0-9]{2})(?<minute>[0-9]{2})_rr1_UX1_EQX.xml
+  time-validity: yearly
+  remote-file-regex:
+    - ".*"
```

Comment:
We added a regex for d2cc. Otherwise this data-bridge scan everything. Can you check ?